overlord/snapstate/snapmgr.go

```diff
@@ -699,6 +699,24 @@ func (snapst *SnapState) InstanceName() string {
 	return snap.InstanceName(cur.RealName, snapst.InstanceKey)
 }
 
+// SnapName returns the name of the snap. Implementation of naming.SnapRef.
+func (snapst *SnapState) SnapName() string {
```

Comment:
maybe it's obvious but where is this used?

Reply:
Hrm, I guess I factored the need for this out at some point. Reverting those commits.